### ISSUE TITLE
Suggestion: HeroesLounge Award System and Player/Team Levels

### ISSUE THREAD

**Issue by phScholz** [kealsera/rikki#43]
You have problems to see if you can trust a new player/sub or if it's a smurf of another player? You want to see immediately if this player or team is new to HeroesLounge or a veteran? You want to give players/users more interaction options with the site and generally increase the identification with the HeroesLounge user or team profile? The suggested HeroesLounge Award system gets you covered. 

**Basics**
	• Introduce awards for various things like "Season 4 participation", "Playoff participation/rank", "Division X end of season rank", "X games played in Heroes Lounge", "x games played as support in Heroes lounge". 
	• Create some shiny symbols for each award which you can show on the players profile in the tab "awards" 
	•  Value each award with points
	• Calculate player level by summing up points of awards
	• Display player level in player profile banner
	• Create awards for teams
	• Display awards on teams page in the award tab
	• Calculate Team level by summing up player level and team award points
	• Show team level in the header of the teams profile page

**possible awards**
	• Season x participation
	• Season x Div y rank z
	• Playoff participation 
	• Playoff rank
	• X games played
	• X games played as "role" 
	• X games played as "hero" 
	• X damage done
	• X healing
	• Highest kda 
	• VIP of the lounge
	• Patreon of the Lounge class x
	• Heroes Lounge twitch subscriber 
	• Heroes Lounge Facebook follower
	• Heroes Lounge Twitter follower
	• Tweeted x times about #hotslounge 
	• Retweeted x times hotslounge tweets
	• Liked x times hotslounge tweets/Facebook posts
	• ... 

**presentation**
	• Display awards on teams and players pages in a specific tab 
	• Display team and player levels in the header of the profiles
	• Display player and team rankings in a subsection of all time rankings 

**benefit**
	• Bind players to their heroes lounge profile and increase identification 
	• Increase interaction with the site
	• Motivate them to share things on social media
	• More exposure for veteran teams and players...

**Comment by kealsera** [comment]
@phScholz That's nothing for this github. That's something for HeroesLounge github as it's not something we just "develop" it has to be thought through written up and content created for it.
You guys have to determine the exact awards you want to give, you have to create all those logos [...] before we can even start developing it.

So please take it to the correct place and open a new issue once everything needed is provided.
